src/components/Input/types.ts

```diff
@@ -0,0 +1,16 @@
+import React, { HTMLAttributes } from 'react'
+import { ThemeType } from '../../types'
+
+export interface InputProps extends HTMLAttributes<HTMLInputElement> {
```

Comment:
I think it is unnecessary to add "undefined" type for non-mandatory properties and error should just be boolean.